src/Minter.sol

```diff
@@ -8,14 +8,15 @@ interface Balot {
   function transferOwnership(address newOwner) external;
 }
 
+// [XXX] Security: either make it ownable either move everything to the constructor
```

Comment:
Yeah so do I understand correctly that someone could frontrun us the moment we gave the minter ownership over the NFT collection, right?